cmd/certinject/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Namecoin Developers GPLv3+
+
+// Command certinject injects certificates into all configured trust stores
+package main
+
+import (
+	"encoding/pem"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/hlandau/xlog"
+	"github.com/namecoin/ncdns/certinject"
+	easyconfig "gopkg.in/hlandau/easyconfig.v1"
+	"gopkg.in/hlandau/easyconfig.v1/cflag"
+)
+
+var log, _ = xlog.New("certinject")
+
+func main() {
+	var (
+		flagGroup = cflag.NewGroup(nil, "certinject")
+		certflag  = cflag.String(flagGroup, "cert", "", "path to certificate to inject into trust store")
+	)
+
+	// read config
+	config := easyconfig.Configurator{
+		ProgramName: "certinject",
+	}
+	config.ParseFatal(nil)
+	cert := strings.TrimSpace(os.ExpandEnv(certflag.Value()))
```

Comment:
Why expand environment variables here?  Wouldn't we expect environment variables to be already expanded by whatever shell spawned the program?

Why trim space?  AFAIK it's perfectly valid for a filename to start or end with a space.

Reply:
so that people can put variables in the config

Reply:
(such as %appdata%  or $HOME)

or should we just use the raw string from config/args ? thats fine too

Reply:
removed trimspace/expandenv in d0e041d